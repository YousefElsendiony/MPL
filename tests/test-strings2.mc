int main()
{
  prints("Mixed Strings and Chars testing!");
  printc('a');

  return 0;
}
