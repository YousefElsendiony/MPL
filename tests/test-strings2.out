"Mixed Strings and Chars testing!"
'a'